go/runtime/rofl/api/bundle_manager.go

```diff
@@ -44,6 +51,8 @@ type BundleAddRequest struct {
 	ManifestHash hash.Hash `json:"manifest_hash"`
 	// Labels are the labels to tag the bundle with so it can later be found.
 	Labels map[string]string `json:"labels"`
+	// Volumes are the volumes to attach to the bundle.
+	Volumes map[string]string `json:"volumes"`
```

Comment:
As mentioned already, this should be passed in a request that provisions a component, if you change how/when components are provisioned.

Reply:
What do you mean? I don't intend to change the BundleAdd API?

Reply:
As mentioned [here](https://github.com/oasisprotocol/oasis-core/pull/6119#discussion_r2057860656), adding bundle to the registry and exploding it should be separated from volume creation and manipulation. Once you have a bundle and volumes in the registry, you should provision a component with the created volumes.

Reply:
I don't think this should be done now as it will introduce further complexity and delay release.